lib/server.js

```diff
@@ -264,7 +264,6 @@ var createSocketIoServer = function(webServer, executor, config) {
     'client store expiration': 0,
     logger: logger.create('socket.io', constant.LOG_ERROR),
     resource: config.urlRoot + 'socket.io',
-    transports: config.transports
```

Comment:
@dhimil Why we remove transports?
New version of socket.io does not allow setup transport?